runners/gradle-plugin/settings.gradle.kts

```diff
@@ -0,0 +1,35 @@
+rootProject.name = "dokka-gradle-plugin"
+
+include(
```

Comment:
I wonder why this is needed - shouldn't the Gradle plugin be treated the same way as any other module/project, or is it different for some reason?

Reply:
Cause `dokka-gradle-plugin` depends on `build-logic`, but root project depends on `dokka-gradle-plugin`.